src/main/java/com/twilio/appointmentreminders/util/TimeZones.java

```diff
@@ -0,0 +1,31 @@
+package com.twilio.appointmentreminders.util;
+
+import java.util.*;
+
+public class TimeZones {
+    private static final String TIMEZONE_ID_PREFIXES =
+            "^(Africa|America|Asia|Atlantic|Australia|Europe|Indian|Pacific)/.*";
+
+    private List<String> timeZones = null;
+    public List<String> getTimeZones() {
+        if (timeZones == null) {
+            initTimeZones();
+        }
+        return timeZones;
+    }
+
+    private void initTimeZones() {
+        timeZones = new ArrayList<String>();
+        final String[] timeZoneIds = TimeZone.getAvailableIDs();
+        for (final String id : timeZoneIds) {
```

Comment:
Can't we use `forEach` here? Although maybe it's easier for people to just use a `for` loop.